src/index.ts

```diff
@@ -0,0 +1,22 @@
+export class Challenge
+{
+    public buzz(n: number) :Promise<any> 
+    {
+        var result: string;
+
+        result = '';
```

Comment:
🙈 A couple of things here:
- since `var` can be accessed from different scopes we prefer to use `let` or `const`. [Here](https://hacks.mozilla.org/2015/07/es6-in-depth-let-and-const/) you can find an in-depth explanation.
- If you assign the value inline, typescript will automatically infer the type for you
```suggestion
        let result = '';
```